### ISSUE TITLE
Bug: basic test not working for Chrome on OSX

### ISSUE THREAD

**Issue by joeblew99** [lucas-clemente/quic-demo#1]
Chrome: Version 50.0.2661.102 (64-bit)
OSX: latest stable

**Bash script as per docs:**

```
$ /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome --user-data-dir=/tmp/chrome --origin-to-force-quic-=quic.clemente.io:443,quic.clemente.io:8005,quic.clemente.io:8006,quic.clemente.io:8007,quic.clemente.io:8008,quic.clemente.io:8009 --enable-quic https://quic.clemente.io
2016-05-15 01:08:52.007 Google Chrome[11513:2346505] NSWindow warning: adding an unknown subview: <FullSizeContentView: 0x7fad89d70710>. Break on NSLog to debug.
2016-05-15 01:08:52.007 Google Chrome[11513:2346505] Call stack:
(
    "+callStackSymbols disabled for performance reasons"
)
May 15 01:12:45  Google Chrome Helper[11561] <Error>: CGAffineTransformInvert: singular matrix.
```

**Screenshot:**

![screencapture-quic-clemente-io-1463267597034](https://cloud.githubusercontent.com/assets/11999859/15271123/7aca7e8c-1a3a-11e6-8b25-29ed12ae798b.png)

**Comment by lucas-clemente** [comment]
Thanks for the report! Could you please double-check the command that you used? I think you are missing the `on` in the flag `origin-to-force-quic-on`?

Also, the demo isn't really "official" yet, we're still working on it (so there might be downtime) and plan to announce it on the QUIC list next week ;)

**Comment by lucas-clemente** [comment]
I've tested it with the Chrome version you're using now, and I'm seeing the same problem. Investigating it :)

**Comment by lucas-clemente** [comment]
I've figured it out. There are 2 separate issues:
- There was a [bug](https://github.com/lucas-clemente/quic-go/issues/101) in our QUIC implementation which prevented Chrome stable from working.
- Chrome stable currently does not support forcing QUIC for multiple origins (which we need for the RTT experiment). I've updated the page at https://quic.clemente.io with separate instructions for stable.
